sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/VersionGenerator.java

```diff
@@ -68,6 +69,18 @@ private static String getJavaVersion() {
         return System.getProperty("java.version");
     }
 
+    private static String getJavaRuntime() {
+        if(isGraalVmNative()) {
+            return "!native";
```

Comment:
do you think it's worth changing `native` -> `graal` or do we prefer to group (any hypothetical future) native runtimes together?

Reply:
makes sense 👍